docs/who-makes-ubuntu/index-roles.md

```diff
@@ -1,42 +1,55 @@
 (roles-and-pathways)=
 # Roles and responsibilities
 
-
-This section will allow us to split out explanation content specifically needed
-by the teams/roles to fulfill their tasks, new members to learn, and for
-prospective members to understand what the role entails
-
-Each of these sections can have a team-specific landing page to act as a
-secondary layer of drop-down menus
-
-## Uploaders
-
-```{toctree}
-:maxdepth: 1
-
-about-roles/index-uploaders
+There are many roles and teams that contribute to Ubuntu development. In general,
+the roles can be sorted according to the level of upload permissions (and
+associated responsibilities). This table summarizes various roles in order of
+increasing upload permissions. The three Archive handling teams have the same
```

Comment:
The table is much better now @s-makin 

But the text no more fits:

step 1 - it is now no more "three"
step 2 - I made you call it special teams which is better already
step 3 - instead of "have the same level of upload permissions" and the potential bikeshedding around that statement let us go with what matters "Members of the special teams are usually core-developers, but have different additional responsibilities." or something like it.